wordpress/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM clearlinux/php-fpm
+MAINTAINER sophia.gong@intel.com
+
+RUN swupd bundle-add curl
```

Comment:
Add a "ARG swupd_args" and then add "--no-boot-update $swupd_args"  to the end of the RUN command adding the curl bundle.  Also, in that same RUN command (on a new line), add a " && rm -rf /var/lib/swupd/*" so we don't waste space hosting metadata files.

Reply:
I think it would be better to add "curl" into the clearlinux/php-fpm image. Because on official docker hub provided php:fpm image, the curl is there. So with the updates on clearlinux/php-fpm, there is no need install "curl" on clearlinux/wordpress.

Reply:
@rustylynch ,@qzheng527
thanks for the comments. 
I'm trying to install curl bundle in php-fpm container, currently there is an issue about curl SSL certification problem (report at https://github.com/clearlinux/dockerfiles/issues/188), after get it resolved. will update this wordpress container patch.